GiniVision/Classes/Extensions.swift

```diff
@@ -151,3 +152,125 @@ internal extension UIDevice {
     }
     
 }
+
+internal extension Collection where Iterator.Element == CFString {
+    
+    var strings: [ String ] {
+        return self.map { $0 as String }
+    }
+    
+}
+
+internal extension NSMutableDictionary {
```

Comment:
This is MetaInformationManager implementation specific and should remain in that file.

Reply:
Moved again as we discussed!